.travis.yml

```diff
@@ -77,13 +90,26 @@ install:
       export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:${DEPS_DIR}/llvm/install/lib"
     fi
 
+  # Install Boost headers
+  - |
+    BOOST_URL="http://sourceforge.net/projects/boost/files/boost/1.60.0/boost_1_60_0.tar.gz"
+    mkdir boost && travis_retry wget --quiet -O - ${BOOST_URL} | tar --strip-components=1 -xz -C boost
```

Comment:
I'd rather keep it cloning from github instead, because otherwise this would require periodic maintenance in order to keep it up to date. If it takes too long, I think even manually cloning the specific submodules we need would be a better idea, even if it could break in the future.